### PR TITLE
LRQA-68818 Forms#PagesCanBeReordered Test Fix

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path [hunk 1]
```diff
@@ -587,13 +587,18 @@
 </tr>
 <!--PAGE-->
 <tr>
-	<td>PAGE_MOVE_ACTIVATED</td>
-	<td>//input[contains(@class,'page-header-title') and @value='${key_fieldName}']/../../../../..//button[contains(@class,'page-button') and @title='${key_movePage}']</td>
+	<td>PAGE_MOVE_UP_ENABLE</td>
+	<td>//input[contains(@class,'page-header-title') and @value='${key_formPageName}']/../../../../..//button[contains(@class,'page-button') and @class='reorder-page-button btn btn-monospaced btn-sm btn-secondary'][1]</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path:591]: @leticiachiu, do you think that we can look for the class `reorder-page-button` only at the end of this path?

The other CSS classes aren't maintained by our team, hence they're not under our control which can make the developers from our team remove them in the future due to changes introduced by who maintain them. Verifying only the class `reorder-page-button` can be a good solution in the long term. :slightly_smiling_face:

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path [hunk 2]
```diff
@@ -587,13 +587,18 @@
...
 </tr>
 <tr>
 	<td>PAGE_MOVE_DISABLED</td>
-	<td>//input[contains(@class,'page-header-title') and @value='${key_fieldName}']/../../../../..//button[@disabled and @title='${key_movePage}']</td>
+	<td>//input[contains(@class,'page-header-title') and @value='${key_formPageName}']/../../../../..//button[@disabled and @class='reorder-page-button btn btn-monospaced btn-sm btn-secondary']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PAGE_MOVE_DOWN_ENABLE</td>
+	<td>//input[contains(@class,'page-header-title') and @value='${key_formPageName}']/../../../../..//button[contains(@class,'page-button') and @class='reorder-page-button btn btn-monospaced btn-sm btn-secondary'][2]</td>
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path:601]: @leticiachiu, do you think that we can look for the class `reorder-page-button` only at the end of this path?

The other CSS classes aren't maintained by our team, hence they're not under our control which can make the developers from our team remove them in the future due to changes introduced by who maintain them. Verifying only the class `reorder-page-button` can be a good solution in the long term. :slightly_smiling_face:

**Review by natocesarrego — CHANGES_REQUESTED** [review]
Good pull! Please see my comments below. :wink:

**Comment by leticiachiu** [comment]
@natocesarrego fixed

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**